core/services/eth/contracts/FluxAggregator.go

```diff
@@ -87,11 +87,11 @@ type FluxAggregatorRoundState struct {
 }
 
 type FluxAggregatorRoundData struct {
-	RoundID         uint32   `abi:"roundId" json:"reportableRoundID"`
+	RoundID         *big.Int `abi:"roundId" json:"reportableRoundID"`
```

Comment:
I ask this a lot ... but here goes again: is a pointer needed here? Is `nil` usefully distinguishable from `0` ?

Reply:
I think to be compatible with the geth generated wrapper it needs to be. See the output of `LatestRoundData()` in `gethwrappers/generated/flux_aggregator_wrapper/flux_aggregator_wrapper.go`

Reply:
I guess we could map the output of the abigen function to any type we want, but just feels like an unnecessary middle step. Open to pushback though!